src/patternfly/components/Card/card-footer.hbs

```diff
@@ -1,3 +1,7 @@
-<div class="pf-c-card__footer {{ pf-c-card__footer-modifier }}">
+<div class="pf-c-card__footer {{#if card-footer--modifier}} {{card-footer--modifier}}{{/if}}"
```

Comment:
I'll just leave this comment once - all of the card `.hbs` files need the leading space  before `{{#if` removed, and for the `>` on the opening tag to be brought up a line after `{{/if}}`